Detour/Source/DetourNavMesh.cpp

```diff
@@ -958,9 +958,9 @@ dtStatus dtNavMesh::addTile(unsigned char* data, int dataSize, int flags,
 		{
 			connectExtLinks(tile, neis[j], -1);
 			connectExtLinks(neis[j], tile, -1);
+			connectExtOffMeshLinks(neis[j], tile, -1);
 		}
 		connectExtOffMeshLinks(tile, neis[j], -1);
-		connectExtOffMeshLinks(neis[j], tile, -1);
```

Comment:
I agree ! I'll try it out tomorrow (it's just moving a line and it should work but I'll test it !).

I'll make the change right after the test.